content/rs/developing/crdbs/developing-hll-crdb.md

```diff
@@ -23,7 +23,7 @@ Here is an example of a simple write case:
 | t2   | --- sync ---      |                   |
 | t3   |                   | PFADD hll x       |
```

Comment:
This command as no effect because `x` already exists in the `hll`. 
Pls change it to `PFADD hll y` and revert the change of the `PFCOUNT` result